### PR TITLE
Update build image to Go version 1.8

### PR DESCRIPTION
**Pull request by cybrcodr**

### CODE REVIEW

**Comment by cybrcodr** [comment]
Just did an update based on our discussion.  PTAL.

**Comment by cybrcodr** [comment]
Another update per feedback from zombiezen@ regarding checksum validation.

File: base/Dockerfile [hunk 1]
```diff
@@ -21,13 +21,13 @@ RUN apt-get update \
        --no-install-recommends \
     && rm -rf /var/lib/apt/lists/*
 
-ENV GO_VERSION 1.6.3
+ENV GO_VERSION 1.8
 
 RUN curl -sSL https://storage.googleapis.com/golang/go$GO_VERSION.linux-amd64.tar.gz -o /tmp/go.tar.gz && \
-    curl -sSL https://storage.googleapis.com/golang/go$GO_VERSION.linux-amd64.tar.gz.sha256 -o /tmp/go.tar.gz.sha256 && \
-    echo "$(cat /tmp/go.tar.gz.sha256)  /tmp/go.tar.gz" | sha256sum -c - && \
+    CHECKSUM=53ab94104ee3923e228a2cb2116e5e462ad3ebaeea06ff04463479d7f12d27ca; \
```
Comment (zombiezen) [inline base/Dockerfile]: s/`;`/`&&`/

Comment (cybrcodr) [inline base/Dockerfile]: Done.

**Pull request merged**